.github/workflows/ci-cassandra.yml

```diff
@@ -34,13 +34,13 @@ jobs:
       uses: step-security/harden-runner@17d0e2bd7d51742c71671bd19fa12bdc9d40a3d6 # v2.8.1
       with:
         egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
-
+        
```

Comment:
exclude this file from the PR, since it's not changing (`git checkout main <file>`)